.github/workflows/list-changed-charts.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
I would add a comment here like: `# Prints a line "${CHART_NAME}_has_changed" for every chart that has changed since last tag`